Sources/Entities/BaseChannel.swift

```diff
@@ -18,7 +18,7 @@ final class BaseChannel<C: PubNubChat.Channel_, M: PubNubChat.Message>: Channel
   var id: String { channel.id }
   var name: String? { channel.name }
   var custom: [String: JSONCodableScalar]? { channel.custom?.mapToScalars() }
-  var description: String? { channel.description_ }
+  var channelDescription: String? { channel.description_ }
```

Comment:
What is a reason behind this change?

Reply:
This is the consequence of applying built-in `CustomStringConvertible` protocol, which allows you to provide a custom string representation of your object, which is particularly useful for debugging.

If your class already had a `description` property, you had to rename it because `CustomStringConvertible` requires a `description` property, which would otherwise cause a conflict.